
        int main;
        void foo(){}
        //error
        